docs/installation/standalone.mdx

```diff
@@ -396,6 +409,17 @@ VelaUX needs authentication. The default username is `admin` and the password is
 
 It requires you to override with a new password for the first login, please make sure to remember the new password.
 
+<details>
+  <summary>possible error and troubleshooting</summary>
+
+  ```
+ Installing VelaUX:
+  Error: "unable to find velaux addon" - This may occur if there is an issue with locating the velaux addon. To resolve this, make sure that the velaux addon is installed properly by running the "vela addon enable ~/.vela/addons/velaux" command. If the issue persists, check the advanced installation arguments for the velaux addon.
```

Comment:
What command will report "unable to find velaux addon"?

Reply:
> make sure that the velaux addon is installed properly by running the "vela addon enable ~/.vela/addons/velaux
This is a bit confusing. VelaD packed a velaux addon with it and unzip it to `~/.vela/addons/velaux` when `velad install`. Advice here should be more clear.

Reply:
> This addon should have been installed automatically by VelaD during its installation process.

This is wrong! addon is unpacked to local directory, not "enabled" (or installed) by `velad install` by default. User can decide to enable it or not. If they do, they can run `vela addon enable ~/.vela/addons/velaux`.

Besides, you said "vela addon enable ~/.vela/addons/velaux" causes this error. You shouldn't ack user to run it again to solve it.